data-extraction/pyDataExtraction/commonTypes/base.py

```diff
@@ -0,0 +1,24 @@
+from json import dumps
+from abc import ABC, abstractmethod
```

Comment:
Is `abstractmethod` used somewhere?

Reply:
no, I didn't realize I had left that in there.